tests/tree.rs

```diff
@@ -3,7 +3,9 @@ use rand::seq::SliceRandom;
 
 #[test]
 fn statics() {
-    const ROUTES: [&str; 11] = [
+    const ROUTES: [&str; 13] = [
+        "",
```

Comment:
Separate this line out, it will be overwritten by `/` and left here they will be randomly broken up and have different values, causing the test to fail.

Reply:
I don't really understand. Are you saying that having `""` and `"/"` is supported but _not at the same time_?

Reply:
There are lines of operations below that will shuffle the array.
`routes.shuffle(&mut rand::thread_rng());`